From: Gal Pressman <galp@mellanox.com>
Date: Mon, 21 Aug 2017 17:54:21 +0300
Subject: net/mlx5: Remove a leftover unused variable
Patch-mainline: v4.14-rc1
Git-commit: 07533c6765de05199417ec73f9c2a495ddd29473
References: bsc#1046305 FATE#322943

mlx5_core_wq is no longer being used and should be removed
from the code.

Signed-off-by: Gal Pressman <galp@mellanox.com>
Signed-off-by: Saeed Mahameed <saeedm@mellanox.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 include/linux/mlx5/driver.h |    2 --
 1 file changed, 2 deletions(-)

--- a/include/linux/mlx5/driver.h
+++ b/include/linux/mlx5/driver.h
@@ -891,8 +891,6 @@ static inline void *mlx5_buf_offset(stru
 		return buf->direct.buf + offset;
 }
 
-extern struct workqueue_struct *mlx5_core_wq;
-
 #define STRUCT_FIELD(header, field) \
 	.struct_offset_bytes = offsetof(struct ib_unpacked_ ## header, field),      \
 	.struct_size_bytes   = sizeof((struct ib_unpacked_ ## header *)0)->field
